mandatory/2-function-creation.js

```diff
@@ -15,17 +20,31 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let conditions = typeof num === "number" && num % 2 === 0 && num <= 100;
```

Comment:
Well done! Another way is to simplify the condition by removing the check if it's a number. ``num % 2 === 0`` will return false if ``num`` is not a number OR not even

Reply:
I am not sure what you mean. Can it get any simpler than this ?